bot/cogs/doc.py

```diff
@@ -150,13 +175,32 @@ def __init__(self, bot: commands.Bot):
         """
         self.base_urls[package_name] = base_url
 
-        fetch_func = functools.partial(intersphinx.fetch_inventory, config, '', inventory_url)
-        for _, value in (await self.bot.loop.run_in_executor(None, fetch_func)).items():
-            # Each value has a bunch of information in the form
-            # `(package_name, version, relative_url, ???)`, and we only
-            # need the relative documentation URL.
-            for symbol, (_, _, relative_doc_url, _) in value.items():
+        package = await self._fetch_inventory(inventory_url, config)
+        if not package:
+            return None
+
+        for group, value in package.items():
+            for symbol, (package_name, _, relative_doc_url, _) in value.items():
```

Comment:
```suggestion
            for symbol, (package_name, _version, relative_doc_url, _) in value.items():
```
Removing the comment was under the idea of not removing any provided info, so the second item in the tuple should still be `_version` to indicate we don't use it (underscore used) and describing the item in case we do need it sometime in future.